addons/base_import/static/tests/import_action_tests.js

```diff
@@ -11,6 +11,7 @@ import {
     patchWithCleanup,
 } from "@web/../tests/helpers/utils";
 import { createWebClient, doAction } from "@web/../tests/webclient/helpers";
+import { dragenterFiles, dropFiles } from "@web/../tests/legacy/utils";
```

Comment:
A bit strange to use legacy helper but I'm guessing those will get converted at some point.

Reply:
This suite is still legacy. @julien-banken this could be the time to convert it to hoot (not in this PR, but after)

Reply:
Ok, we can do that in another PR. As there are no hoot tests for the `import_action` module yet, I've simply added a legacy test to speed things up. I suggest to migrate all of the tests at once when we have more time.